deepchem/rl/a3c.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, value_weight, entropy_weight, **kwargs):
 
   def create_tensor(self, **kwargs):
     reward, action, prob, value = [layer.out_tensor for layer in self.in_layers]
-    log_prob = tf.log(prob)
+    log_prob = tf.log(prob + 0.0001)
```

Comment:
Why 0.0001?  If the goal is just to make sure we never take the log of zero, then machine epsilon seems like a better value.  You can get it as `np.finfo(np.float32).eps`.  Also, might it be better to use `max(prob, eps)` rather than addition?  That will distort the probability distribution less.